docs/installation/config/env_config.rst

```diff
@@ -0,0 +1,284 @@
+.. _installation_env_config:
+
+===================================
+Environment configuration reference
+===================================
+
+Open Zaak can be ran both as a Docker container or directly on a VPS or
+dedicated server. It relies on other services, such as database and cache
+backends, which can be configured through environment variables.
+
+Available environment variables
+===============================
+
+Required
+--------
+
+* ``DJANGO_SETTINGS_MODULE``: which environment settings to use. Available options:
+  * ``openzaak.conf.production``
+  * ``openzaak.conf.staging``
+  * ``openzaak.conf.docker``
+  * ``openzaak.conf.dev``
+  * ``openzaak.conf.ci``
```

Comment:
These need two extra spaces in rst apparently, otherwise it won't render as a sublist:
![image](https://github.com/open-zaak/open-zaak/assets/29249171/a07129bf-86a9-43bb-8fe5-062e4cc75f3d)

```suggestion
    * ``openzaak.conf.production``
    * ``openzaak.conf.staging``
    * ``openzaak.conf.docker``
    * ``openzaak.conf.dev``
    * ``openzaak.conf.ci``
```